test/example/deployment.yaml

```diff
@@ -30,3 +44,6 @@ spec:
               sleep 10 &
               wait  # interrupted quickly on signal.
           done
+        ports:
+        - containerPort: 80
```

Comment:
This can conflict with the ingress port since both are running on the same node.